### ISSUE TITLE
Failures for 2019-03-06 pre-prod XML 1 of 5

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#1531]
pre-prod XML batch 2019-03-06 1 of 5:
https://drive.google.com/open?id=1tLEcr_FPnxFQmaiWLkbdI9a6EsvbYomS

Errors during XSD validation:
```
line 711: element State: Schemas validity error : Element 'State': [facet 'enumeration'] The value '' is not an element of the set {'AL', 'AK', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DE', 'DC', 'FL', 'GA', 'HI', 'ID', 'IL', 'IN', 'IA', 'KS', 'KY', 'LA', 'ME', 'MD', 'MA', 'MI', 'MN', 'MS', 'MO', 'MT', 'NE', 'NV', 'NH', 'NJ', 'NM', 'NY', 'NC', 'ND', 'OH', 'OK', 'OR', 'PA', 'RI', 'SC', 'SD', 'TN', 'TX', 'UT', 'VT', 'VA', 'WA', 'WV', 'WI', 'WY', 'AS', 'FQ', 'GU', 'HQ', 'DQ', 'JQ', 'KQ', 'MH', 'FM', 'MQ', 'BQ', 'MP', 'PW', 'LQ', 'PR', 'VI', 'WQ'}.
```
Line 711 is a State element filled with spaces in a FS240 place of issuance outside USA.

Additional errors during e-QIP validation:
```
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S12-Education: Provide a response for "Telephone number". 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S13A-EmploymentUnemploymentInformation: Provide a response for "Telephone number". 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S13A-EmploymentUnemploymentInformation: Provide a response for "Telephone number for this address". 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S13A-EmploymentUnemploymentInformation: Answer "Do you or did you have an APO/FPO address while at this location?" question yes or no. 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S16-PersonalReferences: Provide one or more valid telephone numbers. 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S16-PersonalReferences: Provide one or more valid telephone numbers. 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S16-PersonalReferences: Provide one or more valid telephone numbers. 
WARN[0003] gov.opm.eqip.ws.webservice.ApplicantDataException: SF86_5_S17-MaritalRelationshipStatus: Provide a response for "Telephone number". 
```

**Comment by ryanhofdotgov** [comment]
Looks like all except one e-QIP validation error (APO/FPO one) are also found in the 2019-03-05 pre-prod batch of XML errors (search for "Failures for 2019-03-05") in open issues.

**Comment by ryanhofdotgov** [comment]
e-QIP validation errors:
* Exception 1 is a duplicate of #1602 
* Exception 2 is a duplicate of #1600  
* Exception 3 is a duplicate of #1603 
* Exception 4 is a duplicate of #1050 (job physical location was listed as Bahamas, which is supposed to trigger APO logic)
* Exceptions 5,6,7 are duplicates of #1530 
* Exception 8 is a duplicate of #1529

**Comment by ryanhofdotgov** [comment]
XSD validation error was erroneously logged against this XML file, when it should have been logged against:
https://drive.google.com/open?id=1qWHKx21dqiiQ6FlsazFhDdEYzRy_ebE4

**Comment by ryanhofdotgov** [comment]
XSD validation error was resolved by #1446 and #1596
